collect_app/src/main/java/org/odk/collect/android/tasks/DownloadFormsTask.java

```diff
@@ -62,7 +60,7 @@ public DownloadFormsTask(FormDownloader formDownloader) {
                 publishProgress(serverFormDetails.getFormName(), currentFormNumber, totalForms);
 
                 formDownloader.downloadForm(serverFormDetails, count -> {
-                    String message = Collect.getInstance().getString(R.string.form_download_progress,
+                    String message = getLocalizedString(Collect.getInstance(), R.string.form_download_progress,
```

Comment:
This is still a localized string but it is actually displayed rather than being used as part of logic.